content/en/apps/guides/hosting/4.x/backups.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Backups in CHT 4.x"
+linkTitle: "Backups"
+weight: 330
```

Comment:
Very specific!

Reply:
Hah! I just wanted to put the self hosted guides up top and the logs and backup on the bottom.  I hit with the `+10` which didn't work, so I hit it with the `+100` which finally did the trick ; )